app/controllers/steps_controller.rb

```diff
@@ -35,7 +35,7 @@ def step_attrs
   end
 
   def current_path(params = nil)
-    step_path(self.class, params)
+    step_path(self.class, params).gsub("%2F", "/")
```

Comment:
Turns out our namespacing causes this weird side effect where the `/` between medicaid and the rest of the path gets encoded to `%2F` so this is here to fix that.